### ISSUE TITLE
Bug in loading CSS into iframe

### ISSUE THREAD

**Issue by rzymek01** [tinymce/tinymce#3298]
[In docs](https://www.tinymce.com/docs/configure/content-appearance/#usingmultiplestylesheetsasarrayexample) regarding `content_css` setting we can read that when providing an array to that setting the URIs can include commas:

> // includes both CSS files in header, ability to have CSS with `,` in URL

Let's consider two code paths:
1) The URI of the page does not end with `#`,
2)The URI of the page ends with `#` .

(Why? There is conditional in `init` method:  `if (!/#$/.test(document.location.href)) { … }` )

ad. 1)
In this case the aforementioned conditional is true and proper link tags will be created as HTML markup and put directly into iframe HTML.
(details: there is `explode` method which checks if given css is an array and return it without exploding by comma)
Everything works here

ad. 2)
In this case, the page URI ends with '#' so the aforementioned conditional will be false, as a result link tags won't be created at this stage of editor initialisation.
The stylesheets are dynamically loaded later...
Now, `tinymce/dom/DOMUtils:loadCss` is playing its role. Each css URI is treated the same, regardlessly if `css_content` is an array or not, _and URI is split by comma which causes errors_.

**To recap**: When page URI ends with `#` and some of the specified `css_content` CSS URIs contain a comma, the error will appear.

**Workaround:**
Commas in CSS URIs can be replaced with `%2C`.

Links:
- https://www.tinymce.com/docs/configure/content-appearance/#usingmultiplestylesheetsasarrayexample

**Comment by rzymek01** [comment]
any update on that?

**Comment by thedrjim** [comment]
This issue has been closed due to inactivity. Tiny will soon be working with members of the community to close other old issues on our GitHub repositories.
Please note the official support window for TinyMCE 4 is ending on December 31, 2020. If you haven't already upgraded to TinyMCE 5 it is highly encouraged that you do so. Upgrading will give you access to the latest releases and updates made to our WYSIWYG editor focused on bringing the latest UX and developer features into the editing experience. To assist with upgrading the editor we have put together a migration guide you can use here - https://www.tiny.cloud/blog/how-to-migrate-from-tinymce-4-to-tinymce-5/
If you have any questions or concerns, you can contact me at community-qa@tiny.cloud. You can also join the discussion on our mailing list at the following link: community-qa+subscribe@tiny.cloud.
